CHANGELOG.md

```diff
@@ -1665,35 +1665,34 @@ This changelog documents all notable changes to VAST and is updated on every rel
 - Evaluation of predicates with negations return incorrect results. For example, the expression `:addr !in 10.0.0.0/8` created a disjunction of all fields to which `:addr` resolved, without properly applying De-Morgan. The same bug also existed for key extractors. De-Morgan is now applied properly for the operations `!in` and `!~`.
 
 ## [0.1] - 2019.02.28
-[0.1]: https://github.com/tenzir/vast/releases/tag/0.1
+[0.1]: <https://github.com/tenzir/vast/releases/tag/0.1>
```

Comment:
Not liking the `<` URL `>` style. You?

Reply:
Me neither, but I'm working on figuring out how not to need this.

Reply:
Maybe disable [MC034](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034)?

Reply:
That did it.